gita/__main__.py

```diff
@@ -397,6 +405,9 @@ def main(argv=None):
                 "and create hierarchical groups based on folder structure.")
     xgroup.add_argument('-b', '--bare', action='store_true',
             help="add bare repo(s)")
+    xgroup.add_argument('-g','--group',
```

Comment:
It should not be in the exclusion group.

Reply:
fixed